tests/python/test_matrix.py

```diff
@@ -19,7 +19,7 @@ def test_python_scope_vector_operations():
 
 
 @ti.host_arch_only
-def test_python_scope_matrix_operations():
+def _test_python_scope_matrix_operations():
```

Comment:
Were these `_` added in order to test the coverage delta? Shall we revert this file?

Reply:
Yes, and as you see, codecov didn't report a failure! We succeed!